Calculator/Calculator/Models/ExpressionParser.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ExpressionParser.swift
+//  Calculator
+//
+//  Created by Charles on 2023/10/08.
+//
+
+enum ExpressionParser {
+    static func parse(from input: String) -> Formula {
+        var formula = Formula(operands: CalculatorItemQueue<Double>(), operators: CalculatorItemQueue<Operator>())
+        let componentsArray = componentsByOperators(from: input)
+        
+        componentsArray.forEach{ component in
+            if let operand = Double(component) {
+                formula.operands.enqueue(operand)
+            } else if let operatorSymbol = Operator(rawValue: Character(component)) {
```

Comment:
Operator 열거형 인스턴스를 만들고 있는데 이름은 왜 Symbol일까요? 💭
operator를 정의했을 때, 예약어 때문에 사용할 수 없었다면 앞 뒤에 `를 붙여 사용할 수 있습니다.
```suggestion
            } else if let `operator` = Operator(rawValue: Character(component)) {
```

Reply:
알려주신 방법으로 수정해보았습니다!